OrbitGl/LiveFunctionsDataView.cpp

```diff
@@ -147,14 +147,22 @@ std::vector<std::string> LiveFunctionsDataView::GetContextMenu(
   bool enable_select = false;
   bool enable_unselect = false;
   bool enable_iterator = false;
-  bool enable_disassembly = !selected_indices.empty();
+  bool enable_disassembly = false;
   const CaptureData& capture_data = GOrbitApp->GetCaptureData();
   for (int index : selected_indices) {
-    const FunctionInfo& function = *GetFunction(index);
-    const FunctionStats& stats = capture_data.GetFunctionStatsOrDefault(function);
-    enable_select |= !GOrbitApp->IsFunctionSelected(function);
-    enable_unselect |= GOrbitApp->IsFunctionSelected(function);
+    const FunctionInfo& selected_function = *GetSelectedFunction(index);
+    const uint64_t absolute_address = FunctionUtils::GetAbsoluteAddress(selected_function);
+
+    // Is that function actually inside module of the process (i.e. can we disassemble?)
+    const FunctionInfo* actual_function =
+        capture_data.FindFunctionByAddress(absolute_address, false);
+    const bool function_exists = actual_function != nullptr;
+
+    const FunctionStats& stats = capture_data.GetFunctionStatsOrDefault(selected_function);
+    enable_select |= !GOrbitApp->IsFunctionSelected(selected_function) && function_exists;
+    enable_unselect |= GOrbitApp->IsFunctionSelected(selected_function) && function_exists;
```

Comment:
Only for conceptual reasons, I would put `function_esists` on the left side of the `&&`.

Reply:
Sure.